include/cpuinfo_ppc.h

```diff
@@ -74,6 +73,11 @@ typedef struct {
 // This function is guaranteed to be malloc, memset and memcpy free.
 PPCInfo GetPPCInfo(void);
 
+typedef struct {
+  char platform[64];       // 0 terminated string
+  char base_platform[64];  // 0 terminated string
+} PlatformType;
```

Comment:
since it is only used by PPC why not renaming it PPCPlatformType or even better merge it
with PPCPlatformStrings struct just below ?

Reply:
That's a good point.

Reply:
Actually I'm still going to keep it as a separate `struct` otherwise it will break user code.

Reply:
done in last commit